### PR TITLE
Wordage update of the data validation.

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

Updated wordage for About The Data data validation. 

**Technical details:**

Grace Asked for the links in the ticket to be actual clickable links.

**JIRA Ticket:**
[DEV-10153](https://federal-spending-transparency.atlassian.net/browse/DEV-10153)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Review by anjenkin — CHANGES_REQUESTED** [review]
See comment from Grace in the ticket

**Comment by chasls2** [comment]
fixed

**Review by anjenkin — CHANGES_REQUESTED** [review]
Grace made another comment that needs to be addressed.  :)

File: src/content/about-the-data/data-validation-of-source-systems.mdx [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+ (Files A, B, and C) as well as the Treasury Financial Assistance Broker 
+ Submission (FABS) system. These validations are based on business rules 
+ (such as for award linkage between File C and Files D1 and D2) and checks 
+ against authoritative sources such as GTAS. Agency financial systems are subject 
+ to other data quality assurance measures, such as review by Offices of Inspector
+  General (OIG) and the Government Accountability Office (GAO) for reports as 
+  required by the DATA Act.
+
+USAspending.gov relies on internal validations of data within source systems 
+outside of the Treasury Department (such as FPDS, FSRS, and  <a target="_blank" rel="noopener noreferrer" href="http://sam.gov/" alt="sam.gov link">SAM.gov</a> | Home).
```
Comment (anjenkin) [inline src/content/about-the-data/data-validation-of-source-systems.mdx:11]: Oh, "| Home" should be removed completely.  It's just an artifact from how JIRA interpreted the link.

Comment (chasls2) [reply src/content/about-the-data/data-validation-of-source-systems.mdx]: My bad. Done! Changing to a 13point ticket. /s

**Review by anjenkin — APPROVED** [review]

**Pull request merged**